site/proxy.md

```diff
@@ -102,7 +112,38 @@ resolution. WeaveDNS is used in addition to any dns servers specified
 via the `--dns` option. More details on weaveDNS can be found in the
 [weaveDNS documentation](weavedns.html).
 
-## Limitations
+## <a name="usage-with-tls"></a>Usage with TLS
+
+If you are using docker with TLS enabled, you probably want to talk to
+the proxy over TLS. Launching the proxy with TLS support is done
+with the same command-line flags as the docker daemon. For example, if
+you have generated your certificates and keys into the docker host's
+`/tls` directory, we can launch the proxy with:
+
+    host1$ weave launch-proxy --tlsverify --tlscacert=/tls/ca.pem \
+             --tlscert=/tls/server-cert.pem --tlskey=/tls/server-key.pem
+
+The paths to your certificates and key, must be provided as absolute
+paths which exist on the docker host.
+
+With the proxy running over TLS, we can configure our regular docker
+client
+
+    host1$ docker --tlsverify --tlscacert=ca.pem --tlscert=cert.pem \
+             --tlskey=key.pem -H=tcp://host1:12375 version
+
+The certificates and keys used by the client will need to be present
+on your client machine.
+
+Because the proxy connects to the docker daemon at
+`unix:///var/run/docker.sock`, you must ensure that the daemon is
+listening there. To do this, you will need to pass the `-H
+unix:///var/run/docker.sock` when booting the docker daemon.
```

Comment:
missing 'option'. also, should say something along the line of "... in addition to the -H options for configuring the TCP listener". Check the docker docs for what the correct terminology is here. And if you can dredge up any docker documentation anywhere that shows multiple `-H`, please refer to that.